NEWS

```diff
@@ -393,6 +393,16 @@ CHANGES WITH 243 in spe:
 
           https://systemd.io/RANDOM_SEEDS
 
+        * When the system is hibernated the swap device to write the
+          hibernation image to is now automatically picked from all available
+          swap devices, preferring the swap device with the highest configured
+          priority over all others, and picking the device with the most free
+          space if there are multiple devices with the highest priority.
```

Comment:
Hmm, does this even make any sense? Swap priority is usually about speed of the device. But the device to swap to should mostly be about size. Dunno, but when looking at this again, I'm not sure if this is the right thing to do.